repository/src/main/resources/db/migration/V20231121_18__GA-18_add_access_type_roles_table.sql

```diff
@@ -0,0 +1,60 @@
+--
+-- Name: access_type_roles; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.access_type_roles (
+                                        id integer NOT NULL,
+                                        live_from date NOT NULL,
```

Comment:
not sure this col needs to be mandatory - JIRA needs updating?

Reply:
Spoke with Suman, live_from is marked as required here: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1712500134#GroupAccessScope-AccessTypeRoles

Reply:
after further discussion agreed to make it non mandatory.